downloads/pt/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: cheat-sheet
+title: Folheto de ajuda para Git do GitHub (pt-BR)
+byline: Git é um sistema de controlo de versão distribuído open source que facilita ações com o GitHub no seu notebook ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.
```

Comment:
This paragraph looks more like pt-BR than pt-PT. I'd rewrite it to: `Git é um sistema distribuído open source de controlo de versões que permite interagir com o GitHub no seu notebook ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.`